CustomerOrderDataBase.java

```diff
@@ -161,9 +164,164 @@ else if (input.contains("d")) {
 
     //This method will have the employee add in customer data into the database
     private void addOrder() {
+        // Variables
+        String date = null, cust_email = null, cust_location = null, product_id = null;
+        int quantity = 0;
+
+        // Date-Time Format "YYYY-MM-DD"
+        DateTimeFormatter dateFormatter = DateTimeFormatter.ISO_LOCAL_DATE;
+        boolean date_Valid = false;
+
+        // Separator for user readability
+        String s = "----------------------------------------"; // separator
+
+        boolean user_confirmed = false;
+        while (!user_confirmed) {
+
+            // module header
+            System.out.println(s);
+            System.out.println("Adding Order: ");
+            System.out.println(s);
+
+            // Getting the user date for entry
+            System.out.print("Enter Date(YYYY-MM-DD): ");
+            date = console.next();
+            //This while loop will check if the date is valid
+            while (!date_Valid) {
```

Comment:
Great job on the validation! It may be useful if we had these validations as their own methods and then we can reuse them from add/update/delete to validate user input